backends/backends.c

```diff
@@ -450,7 +475,18 @@ void *backends_main(void *ptr) {
 
 #if ENABLE_PROMETHEUS_REMOTE_WRITE
     int do_prometheus_remote_write = 0;
-    BUFFER *http_request_header = buffer_create(1);
+    BUFFER *http_request_header = NULL;
```

Comment:
This also looks like a prometheus related change, could you clarify why changed to null?

Reply:
The same reason, as in the previous comment.